src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHDLGeneratorFactory.java

```diff
@@ -172,38 +169,36 @@ public String getComponentStringIdentifier() {
 
   @Override
   public SortedMap<String, Integer> GetInputList(Netlist TheNetlist, AttributeSet attrs) {
-    SortedMap<String, Integer> Inputs = new TreeMap<>();
-    int NrOfClockTrees = TheNetlist.NumberOfClockTrees();
+    final var Inputs = new TreeMap<String, Integer>();
+    final var NrOfClockTrees = TheNetlist.NumberOfClockTrees();
     /* First we instantiate the Clock tree busses when present */
     if (NrOfClockTrees > 0 || TheNetlist.RequiresGlobalClockConnection() || requiresFPGAClock) {
       Inputs.put(TickComponentHDLGeneratorFactory.FPGAClock, 1);
     }
-    for (String in : MyIOComponents.GetMappedInputPinNames()) {
+    for (var in : MyIOComponents.GetMappedInputPinNames()) {
       Inputs.put(in, 1);
     }
     return Inputs;
   }
 
   @Override
   public ArrayList<String> GetModuleFunctionality(Netlist TheNetlist, AttributeSet attrs) {
```

Comment:
variable name should start with lowercase. Please rename all variables and function argument names too.

Reply:
See my remark above